src/app/content/highlights/components/SummaryPopup/HighlightListElement.tsx

```diff
@@ -154,10 +139,10 @@ const HighlightListElement = ({ highlight, locationFilterId, pageId }: Highlight
       onColorChange={updateColor}
     />}
     <HighlightContentWrapper color={highlight.color}>
-      <HighlightContent
+      <ContentExcerpt
         className='summary-highlight-content'
```

Comment:
i believe this class is what applies the content styling. you should move it into `ContentExcerpt` and maybe change the name

Reply:
is `summary-excerpt-styles` fine?

Reply:
why not `.content-excerpt`?